radsat-sk/framework/fileio/RFram.c

```diff
@@ -7,6 +7,7 @@
 #include <RFram.h>
 #include <hal/errors.h>
 #include <hal/Storage/FRAM.h>
+#include <RCommon.h>
 
 

```

Comment:
extra whitespace

Reply:
This is fine since its used to separate a section.

Reply:
https://github.com/USST-RADSAT-SK/software-and-command#newline-whitespace Bottom of Newline Whitespace